common/updates-model.cpp

```diff
@@ -783,6 +783,12 @@ bool updates_model::draw_firmware_section(viewer_model& viewer, const char * win
                 _fw_image.clear();
                 _fw_download_progress = 0;
             }
+            // Verify an error window will not pop up. ImGui cannot handle 2 PopUpModals in parallel.
+            if (!error_message.empty())
+            {
+                LOG_ERROR("error catched during update process, details: " + error_message);
+                error_message.clear();
```

Comment:
What happens then?

Reply:
The error is ignored and the DFU process handle it , if an error occurred during the DFU process it will be in fail state.
Then the SW update window will remain with a failure statue and options to retry or close (After checking the "I understand.." button)

The original code was
if (error_message !="") error_message = "";
I changed the syntax and added a LOG_ERROR message as Evgeni asked.